src/runtime/virtcontainers/kata_agent.go

```diff
@@ -1531,6 +1532,21 @@ func (k *kataAgent) handleLocalStorage(mounts []specs.Mount, sandboxID string, r
 	}
 	return localStorages, nil
 }
+func (k *kataAgent) CheckandSetDmVerityDevice(m Mount) string {
+
+	for _, o := range m.Options {
+		if strings.HasPrefix(o, "extraoption=") {
```

Comment:
How about using `io.katacontainers.volume` instead of `extraoption`?

Reply:
```
/// Key to identify dmverity information in `Storage.driver_options`
pub const KATA_VOLUME_DMVERITY_OPTION_VERITY_INFO: &str = "verity_info";
```
I update the code replacing `extraoption=` with `verity_info=` 
Maybe we should follow the definition in `libs/kata-types/src/volume.rs`